### DISCUSSION TITLE
HA Core

### DISCUSSION THREAD

**Discussion by flauzf** [vanackej/risco-mqtt-local#66] [Q&A]
Hi all
Can it work with home assistant core? without addons section?
thanks in advance

**Comment by vanackej** [comment]
No this is an add-on only, and requires a mqtt broker to run.
To integrate Risco panels without add-on, I would recommend using the official Risco integration with cloud or local mode depending on your needs.

**Reply by flauzf** [reply]
Thank you for your answer.
I have some issues with the official integration. I will try to fix them.
